File: sorting/bubble_s.py
def bubble_sort(arr: list) -> list:
    n = len(arr)
    for x in range(n):
        swapped = False
        for y in range(n-x-1):
            if arr[y] > arr[y + 1]:
                arr[y], arr[y + 1] = arr[y + 1], arr[y]
                swapped = True
        if not swapped:
            break
    return arr


if __name__ == "__main__":
    l = [6, 8, 3, 2, 9, 5]
    m = [9,3,4,5]
    sorted_l = bubble_sort(m)
    print(sorted_l)

File: sorting/bubbleS.js
function bubble_sort(arr){
    let n = arr.length
    for(let i=0; i < n; i++){
        let swapped = false
        for(let j=0; j < n-i-1; j++){
            if (arr[j] > arr[j+1]){
                let temp = arr[j]
                arr[j] = arr[j+1]
                arr[j+1] = temp
                swapped = true
            }
        }
        if(!swapped){
            break
        }
    }
    return arr
}

let l = [2, 3, 9, 1, 6]
console.log(bubble_sort(l))